_info/Tribute

```diff
@@ -1,6 +1,6 @@
 Tribute
 
-Every day, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
+Every day phase, the king has to choose to tribute one of this team members. This is mandatory. The king has to also choose a certain amount of money to assign to the person. Each team will do the same. 
```

Comment:
might be better if you say Each teams king. but technically its any royalty role that can do it, so maybe it would be more accurate to say that

Reply:
Currently only the king and prince can but I'll try to change it soon if the royalty patch goes thought

Also yes Ill update the desc